.github/workflows/env-install/action.yml

```
@@ -13,7 +13,7 @@ runs:
       uses: docker/setup-buildx-action@v3
 
     - name: Install uv
-      uses: astral-sh/setup-uv@v4
+      uses: astral-sh/setup-uv@v5
```

https://github.com/astral-sh/setup-uv/releases/tag/v5.0.1

```suggestion
      uses: astral-sh/setup-uv@180f8b44399608a850e1db031fa65c77746566d3 #v5.0.1
```